src/projects/TodoList/components/CategoryCheckBoxes/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import './style.css';
+
+class CategoryCheckBoxes extends React.Component {
+  render () {
+    // logic goes here
+    let eachCategory = this.props.categoriesInfo.map((category, i) => {
+      let categoryBoxColour = {
+        backgroundColor: category.colour,
+        color: category.text
```

Comment:
You know that conversation we had about using chromajs to set the colour of the text - this is where i'd do it, rather than store a text colour against the category object.

The colour of the text is a view/render concern based on whatever colour the background of the category has been selected as - for that reason I wouldn't keep it in the category object and just have the render here decide what it should look like

Reply:
cooll done!